.gitignore

```diff
@@ -0,0 +1,6 @@
+db.sqlite3
```

Comment:
Oh, that's a good point. I had added db.sqlite3 here (and to my local copy of the other git project I generated during the Djangocon tutorials) because I was adding Django locally and that's its default database. Your project isn't using Django or a DB as such, so there's not much point in having it in the .gitignore. I'll remove it and push a change in a bit.